pkg/validation/strfmt/kubernetes-extensions.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2024 go-swagger maintainers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strfmt
+
+import (
+	"encoding/json"
+	"regexp"
+)
+
+// KubernetesExtensions is the formats registry for JSON Schema formats
+// extensions defined by the Kubernetes project.
+var KubernetesExtensions = NewSeededFormats(nil, nil)
```

Comment:
This is peer to the existing `Default` var? looks like we have a lot of things using that other var... will they need to switch to this one?